### ISSUE TITLE
(not bug, but feature request) Can't use Centos7 Custom AMI in cn-northwest-1 region

### ISSUE THREAD

**Issue by p31415926** [aws/aws-parallelcluster#1500]
**Environment:**
 - AWS ParallelCluster / CfnCluster version [e.g. aws-parallelcluster-2.0.0] 2.5.0
 - OS: [e.g. alinux]  centos7
 - Scheduler: [e.g. SGE]  awsbatch
 - Master instance type: [e.g. m5.xlarge] m5.xlarge
 - Compute instance type: [e.g. c5.8xlarge] c5.xlarge

**Bug description and how to reproduce:**
I tried use custom ami make centos7 ami in cn-northwest-1 region. but looks like pcluster will check amis mapping again. We have already transfer global centos and ubuntu parallelcluster ami 
to cn-north-1 and cn-northwest-1 region. we have customer would like to adopt centos7 to deploy pcluster.

**Comment by demartinofra** [comment]
Can you please share your cluster config? Does it fail if you try to set the imported AMI with the `custom_ami` config option?

**Comment by demartinofra** [comment]
Also marking as duplicate of https://github.com/aws/aws-parallelcluster/issues/1491

**Comment by p31415926** [comment]
Hi Demartinofra,

Yes, I have tried use my custom AMI ID in my cluster config file. but looks like system will still check our region and ami mapping file.

**Comment by demartinofra** [comment]
Can you please share your cluster config so that I can try to reproduce the issue?

**Comment by p31415926** [comment]
I paste my config file.

[aws]
aws_region_name = cn-northwest-1

[global]
cluster_template = default
update_check = true
sanity_check = true

[aliases]
ssh = ssh {CFN_USER}@{MASTER_IP} {ARGS}

[cluster default]
key_name = ZHY
scheduler = awsbatch
master_instance_type = t3.large
vpc_settings = default
custom_ami = ami-01cd28e34080afd53
base_os = centos7

[vpc default]
vpc_id = vpc-03b29e319cfcd50fa
master_subnet_id = subnet-0d91bdee6025358e2

**Comment by demartinofra** [comment]
You are right, currently it is not possible to use custom_ami with an unsupported OS in China, reason being the FindInMap function in the CloudFormation template gets always resolved even if declared in a conditional statement. 

I'll mark this as an enhancement to unblock usage of custom_ami in China.

**Comment by lukeseawalker** [comment]
Following Francesco's answer, it's true that it is not possible to use custom_ami with an unsupported OS in China, but you could use a custom CloudFormation template where you have added the map OS/region that you need. You could map directly your custom AMI and that will permit to avoid to specify a custom_ami parameter in the configuration.

Closing because duplicate of #1491
